lib/src/google_api_availability.dart

```diff
@@ -40,4 +44,100 @@ class GoogleApiAvailability {
 
     return GooglePlayServicesAvailability.values[availability];
   }
+
+  /// Returns true if the device was able to set Google Play Services to available,
+  /// Returns false if the device was unable to set Google Play Services to available or status is unknown.
+  ///
+  /// If it is necessary to display UI in order to complete this request
+  /// (e.g. sending the user to the Google Play store) the passed Activity will be used to display this UI.
+  Future<bool> makeGooglePlayServicesAvailable() async {
+    if (defaultTargetPlatform != TargetPlatform.android) {
+      return false;
+    }
```

Comment:
Since we are breaking the API, I feel it is better to throw an exception for all these method when they are called on a different platform. That way we can provide a consistent API across all methods.

It would be fine to throw the `UnsupportedError` exception:
```suggestion
    if (defaultTargetPlatform != TargetPlatform.android) {
      throw UnsupportedError('Not available on non Android devices.');
    }
```